pkg/kubelet/certificate/certificate_manager.go

```diff
@@ -46,6 +46,9 @@ const (
 // manager. In the background it communicates with the API server to get new
 // certificates for certificates about to expire.
 type Manager interface {
+	// CertificateSigningRequestClient sets the client interface that is used for
+	// signing new certificates generated as part of rotation.
+	CertificateSigningRequestClient(certificatesclient.CertificateSigningRequestInterface)
```

Comment:
nit: prefer `SetCertificateSigningRequestClient`. Noun-style names are usually reserved for getters.